public/index.html

```diff
@@ -16,6 +16,17 @@
           <h2>Game Information</h2>
           <span id="info-text"></span>
       </div>
+      <a href="#" id="debug-btn">Debug - Name</a>
+
+      <div id="debug-panel" class="hidden">
+          <h2>Debug - HW</h2>
+          <a href="#" id="debug-play-btn">Play</a>
+          <a href="#" id="debug-pause-btn">Pause</a>
+          Checkbox: <input type="checkbox" id="aotoReset">
+          <button onclick="aotoReset()">Aoto reset</button>
```

Comment:
```suggestion
          <label for="autoReset" >Auto reset</label>
```

You can use CSS to style the label,  but must use label, not button

You should listen for the `change` event on the surrounding form element, and dispatch on the event.target.name property, or maybe implement a state machine based on `new FormData(form) `

Contact me directly for more info of you need it